skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -562,6 +562,14 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 A recent fix has addressed an issue where some Guest users were being omitted from the output of the Get-CsOnlineUser cmdlet, resulting in an increase in the reported user count.
 
+**Updates in Teams PowerShell Module version 6.1.0 and later**
+
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 6.1.0 or later.
```

Comment:
very confusing line.
"organizations having TeamsOnly users that use Microsoft Teams PowerShell version 6.1.0"

- are you referring to tenant-level setting (`Get-CsTenant | select TeamsUpgradeEffectiveMode`)?

Reply:
@Milehigh-world